lib/kits/core-ui/components/list-publications/main-container.js

```diff
@@ -0,0 +1,94 @@
+import Mustache from 'mustache';
+import {request} from '../../../core';
+import {ComponentHelper, translate} from '../helpers';
+import './main-container.styl';
+
+const defaultTemplate = `\
+<div class="sgn__publications">
+    <ul class="sgn-publications-list-items-container">
+        {{#publications}}
+        <li class="sgn-publications-li">
+            <div class="publications__item" data-id="{{id}}">
+                <div class="sgn-publications-list-content-img">
+                    <img data-id="{{id}}" src="{{images.view}}" alt="{{label}}">
+                </div>
+                <div class="sgn-publications-list-content-text">
+                    <div class="sgn-publications-list-content-heading">
+                        <span>{{label}}</span>
+                    </div>
+                    <div class="sgn-publications-list-content-date">
+                        <span>{{date_from}}-{{date_till}}</span>
+                    </div>
+                </div>
+            </div>
+        </li>
+        {{/publications}}
+    </u;>
+</div>
+<div id="sgn-paged-publication-container"></div\
+`;
+
+const mainContainer = ({configs, template, el}) => {
```

Comment:
This is used a MainContainer everywhere so it's probably best to make that naming consistent

Reply:
All function names are in camel case on declaration only, but they are capitalized when exported. Should we make all function names in capital?

Reply:
I don't feel so strongly about that, as long as the same function doesn't have different names in different places.